main.tf

```diff
@@ -143,6 +143,7 @@ resource "aws_rds_cluster" "this" {
       # See docs here https://registry.terraform.io/providers/hashicorp/aws/latest/docs/resources/rds_global_cluster#new-global-cluster-from-existing-db-cluster
       global_cluster_identifier,
       snapshot_identifier,
+      restore_to_point_in_time,
```

Comment:
I don't think we can naively add this here - what if someone is not using a clone and they want to restore to a point in time? they can't since it will have been ignored